src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import Layout from 'components/Layout';
+import 'styles/App.css';
+import BookList from 'components/BookList';
+import Categories from 'components/Categories';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  const routes = createBrowserRouter([
+    {
+      path: '/',
+      element: <Layout />,
+      children: [
+        { path: 'books', element: <BookList /> },
+        { path: 'categories', element: <Categories /> },
+      ],
+    },
+  ]);
```

Comment:
- [ ] when opening your application on initial load the user gets an empty screen as you can see in below example.
![afbeelding](https://user-images.githubusercontent.com/39240820/230650218-4a5d67b5-4f5f-4bca-a167-23bf03a7e3d6.png)
this is due to the fact that there is not root path defined in your routes, you can solve this issue by making the below adjustment to your code:
```
  const routes = createBrowserRouter([
    {
      path: '/',
      element: <Layout />,
      children: [
        { path: '/', element: <BookList /> },
        { path: 'books', element: <BookList /> },
        { path: 'categories', element: <Categories /> },
      ],
    },
  ]);
```

Reply:
@mikethreels Ok I will do it, Thanks